eventdata/challenges/document_id_benchmark.json

```diff
@@ -165,9 +159,7 @@
             "index.number_of_replicas": 0,
             "index.number_of_shards": 1
           },
-          "mappings": 
-              {% include "mappings.json" %}
-          ,
+          "mappings": "mappings.json",
```

Comment:
Shouldn't this be `../mappings.json` (like the rest)?

Reply:
It should indeed....

Reply:
It should actually be `"mappings.json"`, so have corrected this throughout.